meta-resin-common/recipes-containers/docker/docker_git.bb

```diff
@@ -48,7 +48,10 @@ DEPENDS_append_class-native = " go"
 DEPENDS_append_class-target = " systemd go-cross"
```

Comment:
There is a typo in the commit subject, right? "only link statically link"

Reply:
I dropped this commit in the end. We don't need to statically link the native recipe